apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -195,6 +206,11 @@ function WpcomBlockEditorNavSidebar() {
 			>
 				<div className="wpcom-block-editor-nav-sidebar-nav-sidebar__header">
 					<Button
+						label={ __( 'Close block editor sidebar', 'full-site-editing' ) }
+						showTooltip
+						// We need to shift the focus to something because we are opening a modal
+						// eslint-disable-next-line jsx-a11y/no-autofocus
+						autoFocus
```

Comment:
That's great that `autoFocus` is working!  I think I tried that on another element when I first got started with my [PR](https://github.com/Automattic/wp-calypso/pull/45266/files), but it didn't seem to work,  and then I went down that rabbit whole with programmatic .focus() and `forwardRef`.  

Yay for simplicity.